classes/person.rb

```diff
@@ -1,23 +1,36 @@
-class Person
+require './nameable'
+require '../decorator/decorator'
+
```

Comment:
- [ ] There's a problem with importing classes and I'm getting errors when I run the app. `require` looks for files from the root directory, not the current directory. To go with the current directory you should use `require_relative` or you can assume importing as from the root directory like `require './classes//nameable'` and `require './decorator/decorator'`. Please fix this issue.

![image](https://user-images.githubusercontent.com/83803414/199227790-2161cb14-1ce0-44f9-bb8a-27793e94e707.png)

Reply:
I was in the class folder when trying it I was working but being in the main directory doesn't work unless it's `require_relative`. thank you it's fixed now.